english/make-friends/lol/laugh-out-loud.md

```diff
@@ -3,4 +3,4 @@ You laugh out loud, realizing that the Marshmallow Man was nothing to fear.
 You reach out and shake his hand, saying,
 "It's a shame you don't have opposable thumbs! We won't be able to play thumb wars!"
 
-He chuckles and hands you the lighter, which you help him to light.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.